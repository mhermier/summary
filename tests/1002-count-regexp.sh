#!/bin/sh

utils/generate-test-input.sh 5 100 5 | ./summary -H 5 -F 5 --count --count-skipped --count-matched --regexp 1
