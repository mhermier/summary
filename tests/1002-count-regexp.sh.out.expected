header-1
header-2
header-3
header-4
header-5
entry-1
... 8 entries skipped
entry-10
entry-11
entry-12
entry-13
entry-14
entry-15
entry-16
entry-17
entry-18
entry-19
... 1 entries skipped
entry-21
... 9 entries skipped
entry-31
... 9 entries skipped
entry-41
... 9 entries skipped
entry-51
... 9 entries skipped
entry-61
... 9 entries skipped
entry-71
... 9 entries skipped
entry-81
... 9 entries skipped
entry-91
... 8 entries skipped
entry-100
footer-1
footer-2
footer-3
footer-4
footer-5
20/100 entries matched
